README.md

```diff
@@ -1,3 +1,46 @@
-# Pajarito
+Pajarito
+====
+#### Polyhedral Approximation in Julia: Automatic Reformulations for InTeger Optimization
+
+Pajarito is a mixed-integer convex programming solver package that currently implements an out-of-the-box polyhedral outer approximation algorithm under src/nonlinear.jl and a finite convergence guarantee polyhedral outer approximation and branch and cut for mixed-integer conic programming algorithm under src/conic.jl.
 
 [![Build Status](https://travis-ci.org/mlubin/Pajarito.jl.svg?branch=master)](https://travis-ci.org/mlubin/Pajarito.jl) [![codecov.io](https://codecov.io/github/mlubin/Pajarito.jl/coverage.svg?branch=master)](https://codecov.io/github/mlubin/Pajarito.jl?branch=master)
+
+## Installation
+
+Pajarito can be installed through git:
+
+```
+:~$ git clone https://github.com/mlubin/Pajarito.jl.git
+```
+
+under the include directory of julia. It is recommended to install [ConicNonlinearBridge.jl](https://github.com/mlubin/ConicNonlinearBridge.jl) for the conic programming subproblems in case of numerical instability. See test/ folder for examples.
+
+## Supported solvers
+
+Pajarito requires a mixed-integer linear programming solver and a nonlinear programming solver for src/nonlinear.jl or a conic solver for src/conic.jl. 
+
+For MILP solver, it currently supports only [CPLEX](http://www-01.ibm.com/software/commerce/optimization/cplex-optimizer/) and [GuRoBi](http://www.gurobi.com/) for the branch and cut algorithm for mixed-integer conic programming problems. Pajarito further supports [GLPK](http://www.gnu.org/software/glpk/) for polyhedral outer approximation under both src/nonlinear.jl and src/conic.jl. MILP solver can be specified through the `mip_solver` flag.
```

Comment:
Why doesn't GLPK work for branch and cut? GLPK supports lazy constraint callbacks.